bin/replace_dependencies.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
The fact that we need either to

* add this in the Dockerfile legacy project (7.0, 8.0, not sure which requires it)
* pull OCB after https://github.com/OCA/OCB/commit/b2e48ad8b7cbd62d366e6ffee1861a6085999ce0

Should be documented in the readme (we should have a specific "legacy 7.0/8.0" section in the readme)